infra/dynamodb.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
+  name           = "UserProfiles"
+  billing_mode   = "PAY_PER_REQUEST"
+  hash_key       = "id"
+
+  attribute {
+    name = "id"
+    type = "S"
+  }
+
+  ttl {
+    attribute_name = "TimeToExist"
+    enabled        = true
+  }
+
+  tags = {
```

Comment:
No need for tags currently. But tags are often used for purposes such as billing etc.